src/main/java/com/example/bcsd/Model/Article.java

```diff
@@ -0,0 +1,66 @@
+package com.example.bcsd.Model;
+
+public class Article {
+    private int articleId;
+    private String author;
+    private int boardId;
+    private String title;
+    private String content;
+    private String writeDateTime;
+    private String modifyDateTime;
+
+    public Article() {}
+
+    public Article(int articleId, String author, int boardId, String title, String content, String writeDateTime) {
+        this.articleId = articleId;
+        this.author = author;
+        this.boardId = boardId;
+        this.title = title;
+        this.content = content;
+        this.writeDateTime = writeDateTime;
+    }
+
+    public Article(int articleId, String author, int boardId,
+                   String title, String content,
+                   String writeDateTime, String modifyDateTime) {
+        this.articleId = articleId;
+        this.author = author;
+        this.boardId = boardId;
+        this.title = title;
+        this.content = content;
+        this.writeDateTime = writeDateTime;
+        this.modifyDateTime = modifyDateTime;
+    }
```

Comment:
```suggestion
    public Article(
        int articleId, 
        String author, 
        int boardId,
        String title, 
        String content,
        String writeDateTime, 
        String modifyDateTime
    ) {
        this.articleId = articleId;
        this.author = author;
        this.boardId = boardId;
        this.title = title;
        this.content = content;
        this.writeDateTime = writeDateTime;
        this.modifyDateTime = modifyDateTime;
    }
```

이런식으로 하면 가독성을 좀 더 높일 수 있을것 같아요